views/fragments/listDetections.html

```diff
@@ -100,7 +103,7 @@
                     <!-- Common Name Column -->
                     {{if ne $.QueryType "species"}}
                     <td class="py-1 px-4 font-normal">
-                        <a href="#" hx-get="/note?id={{.ID}}" hx-target="#mainContent" hx-swap="innerHTML"
+                        <a href="#" hx-get="/detections/details?id={{.ID}}" hx-target="#mainContent" hx-swap="innerHTML"
```

Comment:
The '/note?id=...' links cause 404. Hopefully got these right by pointing them to the details view.